series-tvdb.en.yaml

```diff
@@ -2272,6 +2272,17 @@ entries:
         anilist-id: 20993
         start: 13
 
+  - title: "Shaman King"
```

Comment:
On [TVDB](https://www.thetvdb.com/series/shaman-king-2021), the title is "Shaman King (2021)".

Reply:
Fixed